docs/integrations/sources/referralhero.md

```diff
@@ -0,0 +1,27 @@
+# referralhero
+airbyte connector developed using airbyte 1.0 UI Connector development
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| lists | uuid | No pagination | ✅ |  ❌  |
```

Comment:
No pagination is a red flag. 

No link to the actual docs to verify if that's intended.

Reply:
I have included the pagination